sql/glam_etl/fenix_clients_histogram_bucket_counts_v1/query.sql

```diff
@@ -0,0 +1,239 @@
+    histogram_aggregates.metric_type AS metric_type,
+    histogram_aggregates.key AS key,
+    histogram_aggregates.agg_type AS agg_type,
+    aggregates.key AS bucket,
+    aggregates.value
+  FROM
+    normalized_histograms,
+    UNNEST(histogram_aggregates) AS histogram_aggregates,
+    UNNEST(aggregates) AS aggregates
+),
+-- Find information that can be used to construct the bucket range. Most of the
+-- distributions follow a bucketing rule of 8*log2(n). This doesn't apply to the
+-- custom distributions e.g. GeckoView, which needs to incorporate information
+-- from the probe info service.
+-- See: https://mozilla.github.io/glean/book/user/metrics/custom_distribution.html
+distribution_metadata AS (
+  SELECT
+    *
+  FROM
+    UNNEST(
+      [
+        STRUCT(
+          "custom_distribution" AS metric_type,
+          "geckoview_document_site_origins" AS metric,
+          0 AS range_min,
+          100 AS range_max,
+          50 AS bucket_count,
+          "exponential" AS histogram_type
+        ),
+        STRUCT(
+          "custom_distribution" AS metric_type,
+          "gfx_checkerboard_peak_pixel_count" AS metric,
+          1 AS range_min,
+          66355200 AS range_max,
+          50 AS bucket_count,
+          "exponential" AS histogram_type
+        ),
+        STRUCT(
+          "custom_distribution" AS metric_type,
+          "gfx_checkerboard_severity" AS metric,
+          1 AS range_min,
+          1073741824 AS range_max,
+          50 AS bucket_count,
+          "exponential" AS histogram_type
+        ),
+        STRUCT(
+          "custom_distribution" AS metric_type,
+          "gfx_content_frame_time_from_paint" AS metric,
+          1 AS range_min,
+          5000 AS range_max,
+          50 AS bucket_count,
+          "exponential" AS histogram_type
+        ),
+        STRUCT(
+          "custom_distribution" AS metric_type,
+          "gfx_content_frame_time_from_vsync" AS metric,
+          8 AS range_min,
+          792 AS range_max,
+          100 AS bucket_count,
+          "linear" AS histogram_type
+        ),
+        STRUCT(
+          "custom_distribution" AS metric_type,
+          "gfx_content_frame_time_with_svg" AS metric,
+          1 AS range_min,
+          5000 AS range_max,
+          50 AS bucket_count,
+          "exponential" AS histogram_type
+        ),
+        STRUCT(
+          "custom_distribution" AS metric_type,
+          "gfx_content_frame_time_without_resource_upload" AS metric,
+          1 AS range_min,
+          5000 AS range_max,
+          50 AS bucket_count,
+          "exponential" AS histogram_type
+        ),
+        STRUCT(
+          "custom_distribution" AS metric_type,
+          "gfx_content_frame_time_without_upload" AS metric,
+          1 AS range_min,
+          5000 AS range_max,
+          50 AS bucket_count,
+          "exponential" AS histogram_type
+        )
+      ]
+    )
+  UNION ALL
+  SELECT
+    metric_type,
```

Comment:
Would the same metric type appear twice in `distribution_metadata` if it was a `custom_distribution`? `unnested` doesn't seem to have them filtered out.

Reply:
Good catch, it looks like this might be the case.